### DISCUSSION TITLE
Contract is locked by another party

### DISCUSSION THREAD

**Discussion by mauroinagoki** [siacentral/host-manager#93] [Support] [answered]
Why my sia Host is getting error?:

Host misconfigured
Other-benchmark: Unespecified error during an upload/download benchmark

Full details: "NewSession: Lock: contract is locked by another party"

**Comment by n8mgr** [comment, marked as answer]
That is not an issue with your host but with SiaStats. SiaStats has been having weird problems recently. They need to fix it on their end; there's nothing you need to do.
